cmd/migrate.go

```diff
@@ -130,15 +131,26 @@ func migrateReset() error {
 		return err
 	}
 
-	logger.Info("migration executed successfully")
+	logger.Info("migrations reset successfully")
 	return nil
 }
 
 func initMigrations(vipr *viper.Viper, logger log.Logger) (*migrate.Migrate, error) {
-	pgCfg := flags.NewPostgresConfig(vipr)
+	pgCfg, err := flags.NewPostgresConfig(vipr)
+	if err != nil {
+		errMessage := "failed to extract postgres configuration"
+		logger.WithError(err).Error(errMessage)
+		return nil, err
+	}
+
+	dbURL := fmt.Sprintf("postgres://%s:%s@%s:%s/%s?", pgCfg.User, pgCfg.Password, pgCfg.Host, pgCfg.Port, pgCfg.Database)
+	params := url.Values{}
+	params.Add("sslmode", pgCfg.SSLMode)
```

Comment:
If SSLMode is `disabled`, does this going to work? I would suggest to set params depending on the mode being selected such as follow
```
switch pgCfg.SSLMode{
case  "verifyFull :
// Do what ever...
case "verfifyCa:
///
....
```

I think you can also get inspired by our Dialer https://github.com/ConsenSys/quorum-key-manager/blob/main/src/infra/postgres/client/dialer.go#L21

Reply:
Nothing should be done on our end as the package will handle everything. If you pass sslmode=disable and you also pass certificates, they will be ignored. If you pass sslmode=verify-ca without certificates it will fail etc. We don't need that switch or any other logic